### ISSUE TITLE
Missing testbed.exe & instant-ngp.exe

### ISSUE THREAD

**Issue by Eamon-q** [NVlabs/instant-ngp#1444]
Hi,

Really excited to try this out and I have been working through the instructions to get everything working.

Everything seems to have worked so far and I have managed to run a series of images through COLMAP and now have the json file next to my images ready for the next step.

When I run:
C:\ngp\instant-ngp\build\testbed.exe --scene data/image-set-name

I get this back:
'C:\ngp\instant-ngp\build\testbed.exe' is not recognized as an internal or external command,
operable program or batch file.

I noticed that there is no testbed.exe file in the build folder and saw, (in these discussions) that it was already suggested to someone else with the same issue, to use the instant-ngp.exe file in the root folder instead. However this does not work for me and when i look, there is also no instant-ngp.exe file there.

Will really appreciate your help.

Thanks!

Eamon

**Comment by monviso** [comment]
Hi, I have the same issue. No testbed.exe file
Hope for some ideas!

Thanks

**Comment by Eamon-q** [comment]
Hi,

In the meantime I solved this. Firstly, you no longer need "testbed.exe", but instead should look for "instant-ngp.exe". However in my instance I was also missing the instant-ngp.exe file. If you don't have the .exe file then something has gone wrong when running the CMake build. I think for me I was using the wrong python version. If you get stuck, let me know and I'll have a look back through my notes.

Good luck!

**Comment by monviso** [comment]
Thanks a lot for this. I do not have the instant-ngp.exe file as well. 
Actually, I had some problem running the cmake build and I ended up with this specs:
`D:\tutorial\ngp\instant-ngp>cmake -D CUDA_TOOLKIT_ROOT_DIR=D:\tutorial\NVIDIAGPUComputingToolkit\CUDA\v11.6\bin\ncvv.exe -D TCNN_CUDA_ARCHITECTURES=61 -D CMAKE_CUDA_COMPILER=D:\tutorial\NVIDIAGPUComputingToolkit\CUDA\v11.6\bin\ncvv.exe`

where I had to remove the `-B build` part otherwhise I was running into the following error: 

CMake Error at dependencies/tiny-cuda-nn/dependencies/fmt/CMakeLists.txt:234 (target_compile_features):
  target_compile_features no known features for CXX compiler

  "MSVC"

Could this be the issue?

Nonetheless, without `-B build`, it seems to work fine despite having some warnings such as: 
-CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.

--- Could NOT find Vulkan (missing: Vulkan_LIBRARY Vulkan_INCLUDE_DIR) (found version "")
CMake Warning at CMakeLists.txt:117 (message):
  Vulkan was not found.  Instant neural graphics primitives will still
  compile and run correctly, but DLSS will not be supported.

--- Found PythonInterp: D:/miniconda3/python.exe (found suitable version "3.10.9", minimum required is "3")
-- Found OpenGL: opengl32
-- Enabling OpenGL support
-- Could NOT find Vulkan (missing: Vulkan_LIBRARY Vulkan_INCLUDE_DIR) (found version "")
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
-- Could NOT find JsonCpp (missing: JsonCpp_INCLUDE_DIR JsonCpp_LIBRARY)
-- Could NOT find glslc, using precompiled .spv files
-- Looking for secure_getenv
-- Looking for secure_getenv - not found
-- Looking for __secure_getenv
-- Looking for __secure_getenv - not found
-- Looking for timespec_get
-- Looking for timespec_get - found
-- Found and will use pre-generated xr_generated_dispatch_table.h in source tree
-- Found and will use pre-generated xr_generated_dispatch_table.c in source tree
-- Found and will use pre-generated xr_generated_loader.hpp in source tree
-- Found and will use pre-generated xr_generated_loader.cpp in source tree
-- Using Win32 for window creation
-- Found OpenMP_C: -openmp (found version "2.0")
-- Found OpenMP_CXX: -openmp (found version "2.0")
-- Found OpenMP: TRUE (found version "2.0")
-- Could NOT find Python (missing: Python_EXECUTABLE Python_LIBRARIES Python_INCLUDE_DIRS Interpreter Development Development.Module Development.Embed) (Required is at least version "3.7")
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of off64_t
-- Check size of off64_t - failed
-- Looking for fseeko
-- Looking for fseeko - not found
-- Looking for unistd.h
-- Looking for unistd.h - not found


all these sound a bit black boxes for me... I am pretty new to all this.
Many thanks for your help!!
